lib/src/mt_ptp.c

```diff
@@ -115,9 +119,21 @@ static void ptp_coeffcient_result_reset(struct mt_ptp_impl* ptp) {
   ptp->coefficient_result_cnt = 0;
 }
 
+#ifdef MT_PTP_USE_PI
+static void ptp_update_coefficient(struct mt_ptp_impl* ptp, int64_t error) {
+  if (error > 1000 * 1000) return;
+  ptp->integral += (error + ptp->prev_error) / 2;
+  ptp->prev_error = error;
+  double u = MT_PTP_KP * error + MT_PTP_KI * ptp->integral;
+  if (ptp->t2_mode == MT_PTP_L4) u /= 4; /* sync interval is 0.25s for l4 */
```

Comment:
Can we assume sync interval is 0.25s?

Reply:
this interval is the default in our environment, maybe it can be detected/ set by user.